portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,176 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>My Portfolio 1</title>
+
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital@1&display=swap" rel="stylesheet">
+    
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <nav> 
+          <span>Navbar</span>
+            <a href="#box">Color Picker</a>
+            <a href="#gallery">Gallery</a>
+            <a href="#aboutContent">About</a>
+      </nav>
+
+    <div class="hero"> 
```

Comment:
Hero seems a bit presumptuous :)

Reply:
lol no the header image thing is called a hero image header because it was called that on the w3schools documentation.